src/Angor/Client/Pages/Create.razor

```diff
@@ -492,207 +497,270 @@
         </div>
 
         <div class="info-card">
+            <div class="text-center mb-4">
+                <h4 class="mb-2">Deploy Your Project</h4>
+                <p class="text-muted mb-0">Complete the deployment process to make your project live</p>
+            </div>
 
+            @if (!hasErrors && !isManualMode)
+            {
+                <div class="text-center mb-4">
+                    <div class="info-card d-inline-flex align-items-center p-2">
+                        <Icon IconName="lightning" Width="16" Height="16" Class="me-2 text-success" />
+                        <span class="text-success">Automatic Mode</span>
+                        <button class="btn btn-sm btn-border-warning ms-2" @onclick="SwitchToManualMode">
+                            Switch to Manual
+                        </button>
+                    </div>
+                </div>
+            }
+            else if (isManualMode && !hasErrors)
+            {
+                <div class="text-center mb-4">
+                    <div class="info-card d-inline-flex align-items-center p-2">
+                        <Icon IconName="settings" Width="16" Height="16" Class="me-2 text-primary" />
+                        <span class="text-primary">Manual Mode</span>
+                        <button class="btn btn-sm btn-border-warning ms-2" @onclick="ReturnToAutoMode" disabled="@isAutoDeploying">
+                            Return to Auto
+                        </button>
+                    </div>
+                </div>
+            }
+            else if (hasErrors)
+            {
+                <div class="text-center mb-4">
+                    <div class="d-inline-flex align-items-center p-2 rounded border border-warning">
+                        <Icon IconName="warning" Width="16" Height="16" Class="me-2 text-warning" />
+                        <span class="fw-semibold text-warning">Manual Mode Required</span>
```

Comment:
I dont think this is what I had in mind, basically we just need one button that say continue and if we failed we do the prev step again right? so it should always be automatic mode